jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/target_systems/KafkaIntegrationTest.java

```diff
@@ -371,7 +371,20 @@ void endToEnd() {
       List<Consumer<Metric>> assertions = new ArrayList<>(kafkaBrokerAssertions());
       assertions.addAll(
           Arrays.asList(
-              metric -> assertGauge(metric, "jvm.classes.loaded", "number of loaded classes", "1"),
+              metric ->
+                  assertSum(
+                      metric,
+                      "process.runtime.jvm.classes.loaded",
+                      "Number of classes currently loaded",
+                      "{classes}",
+                      false),
+              metric ->
+                  assertSum(
+                      metric,
+                      "process.runtime.jvm.classes.unloaded",
+                      "Number of classes unloaded since JVM start",
+                      "{classes}",
+                      false),
```

Comment:
@Mrod1598 @rmfitzpatrick @dehaansa do you think there's value in re-asserting the JVM metrics in the Kafka test, or does it make sense to remove these assertions?

Reply:
This is intended to test the multiple target system support so requires some redundancy by design: https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/src/integrationTest/resources/target-systems/jvm-and-kafka.properties#L4